app/views/shared/_improve_this_page.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="improve-this-page" data-module="improve-this-page">
+  <div class="js-prompt">
+    <h3 class="improve-this-page__is-useful-question">Is this page useful?</h3>
+    <%= link_to contact_govuk_path, { class: 'button js-page-is-useful', role: 'button' } do %>
+      Yes <span class="visually-hidden"> this page is useful</span>
+    <% end %>
+    <%= link_to contact_govuk_path, { class: 'button js-offer-feedback', role: 'button' } do %>
+      No <span class="visually-hidden"> this page is not useful</span>
+    <% end %>
+    <%= link_to contact_govuk_path, { class: 'button js-offer-feedback improve-this-page__anything-wrong', role: 'button' } do %>
+      Is there anything wrong with this page? <span class="visually-hidden"> this page is not useful</span>
+    <% end %>
+  </div>
+  <div class="js-feedback-form js-hidden">
+    <div class="grid-row">
+      <div class="column-two-thirds">
+        <div class="js-errors"></div>
+        <form>
+          <input type="hidden" name="path" value="<%= request.original_url -%>">
+          <input type="hidden" name="user_agent" value="<%= request.user_agent -%>">
```

Comment:
I need to better understand what ERB escapes when, so forgive me if this is a stupid question – if the user agent or the original url contain quotes or other special characters I'm presuming these'll be escaped as HTML entities?